modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java

```diff
@@ -421,6 +444,41 @@ protected Instance testGetProcessInstancesPage_addInstance(
 		return instance;
 	}
 
+	protected Instance testGetProcessInstancesPage_addInstance(
```

Comment:
@seltong, this method have some contents duplicated in this [other method](https://github.com/liferay-workflow/liferay-portal/pull/452/commits/fee14b6bfd75004b500afc4ee56ad44d0a0f0458#diff-a74bcccf83965baf2e916a5d957a8244f06e9973aa2e5f66224950a882b875b1R419), could you join both?